woocommerce/woocommerce
# Fix `TableCard`s menu being displayed off the page

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





- [Forward placement prop from EllipsisMenu to the Dropdown](https://github.com/woocommerce/woocommerce/commit/6df34ae3e72f8d89b8fd9df7acd698fa7755c9e6)
- [Place ellipsis popover at bottom-end of TableCard's menu.](https://github.com/woocommerce/woocommerce/commit/8cc9bd171e690ba43e8b9d751e79d3b27c4054db) 

Fixes https://github.com/woocommerce/woocommerce/issues/43256



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout and build this branch
2. Run automated tests
3. Go to 
   - _Analytics > Revenue_ `/wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Frevenue`
   - _Analytics > Dashboard_ `/wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Foverview`
5. Click on the kebab icon(s)
6. Check that the menu is displayed within the viewport.
8. Smoke test other instances of Ellipsis menu for the regression, like inbox panel on the dashboard `/wp-admin/admin.php?page=wc-admin`

#### Before
![image](https://github.com/user-attachments/assets/ae451013-9f71-414b-9717-e48fbcdf8a68)


#### After
![image](https://github.com/user-attachments/assets/1c6adfa2-f81f-40e9-a261-48fa25d55d83)




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
